Makefile

```diff
@@ -406,7 +406,7 @@ clean-test: ## clean up debris from previously failed tests
 	@rm -rf bin/cni-bridge-fp
 	@rm -rf bin/containerd-shim-runc-fp-v1
 
-install: ## install binaries
```

Comment:
I don't think so. install is here to install binaries. why I have to install install-man here?

Reply:
The general consensus for decades is that 'install' rule installs the whole package, not just some pieces. On *nix platforms, manpages (describing commands) usually belong together with the command binaries.

Reply:
I think install-man can fix your package issue. In go, most of projects only install the binary. Different project has different convention.